### PR TITLE
increase test buffer size to account for edge case

### PR DESCRIPTION
**Pull request by JacobBarthelmeh**

### CODE REVIEW

File: tests/api.c [hunk 1]
```diff
@@ -7791,7 +7791,9 @@ static int test_wc_RsaKeyToDer (void)
     WC_RNG  rng;
     byte*   der;
 
-    der = (byte*)XMALLOC(610, NULL, DYNAMIC_TYPE_TMP_BUFFER);
+    /* (2 x 128) + 2 (possible leading 00) + (5 x 64) + 5 (possible leading 00)
+       + 3 (e) + 8 (ASN tag) + 10 (ASN length) + 4 seqSz + 3 version */
+    der = (byte*)XMALLOC(611, NULL, DYNAMIC_TYPE_TMP_BUFFER);
```
Comment (dgarske) [inline tests/api.c]: Does it make sense to add a define or enum in rsa.h for this number?

Comment (JacobBarthelmeh) [reply tests/api.c]: Possibly but portions of it are going to be changing due to RSA key size. Both the 128 and 64 values are derived from key size and the size of e could potentially vary too.
